src/definitions/elements/step.less

```diff
@@ -164,6 +166,113 @@
             Types
 *******************************/
 
+& when (@variationCircularSteps) {
+    .ui.circular.steps {
+        border: 0;
+        border-radius: 0;
+        box-shadow: none;
+
+        &:not(.vertical) {
+            background: transparent;
+            border: 0;
+            border-radius: 0;
+            box-shadow: none;
+            position: relative;
+            display: flex;
+            padding: @circularStepsMargin;
+            margin: @circularStepsPadding;
+
+            & .step {
+                background: @circularStepColor;
+                cursor: default;
+                height: @circularStepBorderWidth;
+                display: flex;
+                flex-wrap: wrap;
+                align-items: center;
+                flex: 1;
+                position: relative;
+                margin-left: @circularStepLineMargin;
+
+                &::before {
+                    left: -@circularStepLineMargin;
+                }
+                &::after {
+                    left: @circularStepIconDistance;
+                }
+
+                &.link ::before,
+                &.link ::after {
+                    cursor: pointer;
+                }
+                &.active,
+                &.completed {
+                    background: @circularStepCompletedBackground;
+                }
+                &.completed::after {
+                    left: calc(-@circularStepLineMargin + (calc((@circularStepLineMargin / 2) / 2)));
```

Comment:
You are right. `em` always gives the same result regardless of the size. Let me refactor the `calc` that includes only `em` unit.

`left: calc(-@circularStepLineMargin + (calc((@circularStepLineMargin / 2) / 2)) + 1px);`

We cannot use the fixed value `1px` which can bring different result when the value of `@circularStepLineMargin` has changed. I thought that should be based on the half amount of `@circularStepBorderWidth`.

What do you think?

Reply:
That additional pixel are now evaluated from `@circularStepBorderWidth`. The calc() with em units are kept, since they are evaluating with the variables which could be override by the user, where the default value of `2em / 2` and `4em / 2` in user customized variables would be different outcomes.